### PR TITLE
[luci] Introduce FakeQuant IR

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce CircleFakeQuant IR for FAKE_QUANT Op.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #5610, from draft #5625

File: compiler/luci/lang/include/luci/IR/Nodes/CircleFakeQuant.h [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  int32_t num_bits(void) const { return _num_bits; }
+  void num_bits(int32_t num_bits) { _num_bits = num_bits; }
+
+  bool narrow_range(void) const { return _narrow_range; }
+  void narrow_range(bool narrow_range) { _narrow_range = narrow_range; }
+
+private:
+  float _min = 0.0f;
+  float _max = 0.0f;
+  int32_t _num_bits = 0;
```
Comment (jinevening) [inline compiler/luci/lang/include/luci/IR/Nodes/CircleFakeQuant.h:54]: (optional) how about following the default values in TF?

data| type
-- | --
min | An optional float. Defaults to -6.
max | An optional float. Defaults to 6.
num_bits | An optional int. Defaults to 8.
narrow_range | An optional bool. Defaults to False.

**Review by jinevening — APPROVED** [review]

File: compiler/luci/lang/include/luci/IR/Nodes/CircleFakeQuant.h (hunk 1 above)
Comment (seanshpark) [reply compiler/luci/lang/include/luci/IR/Nodes/CircleFakeQuant.h:54]: Thanks! Current values are from schema generated file (as it's circle file).
I don't think we had default value policy (usually values were 0) but we can decide one.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**